js/interface.js

```diff
@@ -794,6 +794,22 @@ function save(origin, submission) {
     });
 }
 
+function checkFileExtention (fileName) {
+  var regexp = /[^.]+$/;
+  var isCorrectExtension = fileName.match(regexp)[0];
```

Comment:
`.match()` can return `null`, in which case your code will fail.

Reply:
@squallstar 
When we press the cancel button, should we show a user an error message?
Because when we press the cancel button, we also will receive this error message now.
I have tested this code on this type of file: image.jpg, image.png, .gitignore and it worked just fine.

Reply:
@YaroslavOvdii 
- try with a file not having an extension
- pressing cancel should not take any action

Reply:
@squallstar Fixed